### DISCUSSION TITLE
Show User Registration,Login Settings while not having the permission ManageUsers

### DISCUSSION THREAD

**Discussion by Mordsspass** [OrchardCMS/OrchardCore#11446] [Q&A] [answered]
Is it possible and if how can someone enable Users with specific Permissions lets say View Users, who don't have the Manage Users permission to see and edit the settings for Login,Registration and so on.
Background :  I want to enable a role to change the registration settings but not to manage users with the role admin ( which makes it impossible to use Manage Users ) 
I tried overwriting the used Controllers (OrchardCore.Core.Users ), which works for displaying the admin menu section, but doesn't work for the page content. 

Is there an option besides customizing a new module?

**Comment by ns8482e** [comment, marked as answer]
The permission is verified in `LoginSettingsDisplayDriver`  you can create your on driver that replace this driver with custom permission check that you introduce

**Reply by Mordsspass** [reply]
Thanks, was apparently on both eyes blind, searched the whole module but didn't see it.
